components/checkbox.js

```diff
@@ -80,13 +80,21 @@ const mobileLabelStyle = css`
     font-size: 16px;
   }
 `;
+const sidebarLabelStyle = css({
+  fontSize: "14px !important",
+  padding: "6px 10px 15px 12px !important"
+});
 
-const Checkbox = ({ children, className, ...props }) => {
+const Checkbox = ({ children, className, sidebar, ...props }) => {
   const guid = uuidv4();
   return (
     <StyledCheckbox className={className} htmlFor={guid}>
       <StyledInput type="checkbox" {...props} id={guid} />
-      <StyledLabel className={mobileLabelStyle}>{children}</StyledLabel>
+      <StyledLabel
+        className={cx(mobileLabelStyle, sidebar ? sidebarLabelStyle : null)}
```

Comment:
I was pretty happy about this line for conditionally changing the font size and padding in the sidebar multiple choice options + needs options.